packages/components/src/BulletListItem/index.ts

```diff
@@ -0,0 +1 @@
+export { ListItemProps, BulletListItem } from './BulletListItem';
```

Comment:
```suggestion
export * from './BulletListItem';
```
из компонента экспортируем все